packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1273,6 +1273,13 @@ export default class Meeting extends StatelessWebexPlugin {
   setUpLocusInfoMeetingListener() {
     this.locusInfo.on(EVENTS.REMOTE_ANSWERED_DECLINED, (payload) => {
       this.meetingFiniteStateMachine.remote(payload);
+
+      if (payload.shouldLeave) {
+        this.leave({reason: payload.reason})
```

Comment:
leave the call by default

Reply:
Do you have a better message after the the leave promised is fulfilled?

`Meeting:index#setUpLocusInfoMeetingListener --> REMOTE_ANSWERED_DECLINED. ...`